appengine/standard-cloud-tasks/app.rb

```diff
@@ -0,0 +1,33 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# [START cloud_tasks_appengine_standard_quickstart]
+STDOUT.sync = true    # Recommended to reduce log transmission delays
```

Comment:
It looks like rubocop is flagging this. Do not use more than one space between the code and comment.

Reply:
Done